### DISCUSSION TITLE
How I can define path to file

### DISCUSSION THREAD

**Discussion by FedyaSavchuk** [mockoon/mockoon#528] [Help] [answered]
How I can define path to file, without writing absolute path. Could I use png bytecode in response?

**Comment by 255kb** [comment, marked as answer]
I would say, for PNG, the easiest would be to use a file. Never tested to copy the file content in the body editor. I guess it wouldn't work due to the stringification...
For the path, you can specify a relative one. Only tradeoff is that it will be relative to Mockoon's executable. But you can "navigate" using `../../`. 
This will likely change in a near future. I am currently working on the [new storage system](#16) where you can open environment files from anywhere on your hard drive. The file path will be relative to this file and not anymore to Mockoon's executable. Like this it will be easier to share among coworkers.
